specs/lib/bakeAmbientOcclusionSpec.js

```diff
@@ -0,0 +1,525 @@
+                    }
+                }
+            }
+        },
+        "scene": "defaultScene",
+        "scenes": {
+            "defaultScene": {
+                "nodes": [
+                    "squareNode"
+                ]
+            }
+        },
+        "nodes": {
+            "squareNode": {
+                "children": [],
+                "matrix": [
+                    2, 0, 0, 0,
+                    0, 2, 0, 0,
+                    0, 0, 2, 0,
+                    0, 0, 0, 1
+                ],
+                "meshes": [
+                    "mesh_square"
+                ],
+                "name": "square",
+                "extras": {
+                    "_pipeline": {}
+                }
+            }
+        },
+        "meshes": {
+            "mesh_square": {
+                "name": "square",
+                "primitives": [
+                    {
+                        "attributes": {
+                            "POSITION": "accessor_position",
+                            "NORMAL": "accessor_normal",
+                            "TEXCOORD_0": "accessor_uv"
+                        },
+                        "indices": "accessor_index",
+                        "extras": {
+                            "_pipeline": {}
+                        }
+                    }
+                ]
+            }
+        }
+    };
+
+
+
+    beforeAll(function(done) {
+        fs.readFile(boxGltfPath, function(err, data) {
+            if (err) {
+                throw err;
+            }
+            else {
+                boxGltf = JSON.parse(data);
+                addPipelineExtras(boxGltf);
+                loadGltfUris(boxGltf, loadGltfUriOptions, function(err, gltf) {
+                    if (err) {
+                        throw err;
+                    }
+                    done();
+                });
+            }
+        });
+
+        fs.readFile(boxOverGroundGltfPath, function(err, data) {
+            if (err) {
+                throw err;
+            }
+            else {
+                boxOverGroundGltf = JSON.parse(data);
+                addPipelineExtras(boxOverGroundGltf);
+                loadGltfUris(boxOverGroundGltf, loadGltfUriOptions, function(err, gltf) {
+                    if (err) {
+                        throw err;
+                    }
+                    done();
+                });
+            }
+        });
+    });
+
+    // tetrahedron
+    var point0 = new Cartesian3(0.0, -1.0, 1.0);
+    var point1 = new Cartesian3(1.0, -1.0, -1.0);
+    var point2 = new Cartesian3(-1.0, -1.0, -1.0);
+    var point3 = new Cartesian3(0.0, 1.0, 0.0);
+
+    var tetrahedron = [
+        {positions: [point0, point1, point2]},
+        {positions: [point0, point1, point3]},
+        {positions: [point1, point2, point3]},
+        {positions: [point2, point0, point3]}
+    ];
+
+    function testContainmentAndFitCartesian3(min, max, cartesian3s) {
```

Comment:
Ah. This is left over from when I was generating that `texelPoint` cache, it's not used anymore.